sklearn/ensemble/tests/test_bagging.py

```diff
@@ -663,3 +663,28 @@ def test_oob_score_removed_on_warm_start():
     clf.fit(X, y)
 
     assert_raises(AttributeError, getattr, clf, "oob_score_")
+
+
+def test_consistent_index_sampling():
+    # Make sure randomly drawn indices generated at fit time are identically 
+    # reproduced at a later time (for use in, e.g., OOB scoring)
+    X, y = make_hastie_10_2(n_samples=20, random_state=1)
+    
+    bagging = BaggingClassifier(KNeighborsClassifier(), max_samples=0.5, max_features=0.5)
+    bagging.fit(X, y)
+
+    feat_inds_list = bagging.estimators_features_
+    feat_samp_inds_gen = bagging._get_estimators_data_draws()
```

Comment:
It can be dynamically generated any time after `fit()`, but I want to make sure the exact values that were generated during `fit()` (which, for `estimators_features_`, are still stored on the object) can be reproduced at a later time.

Since the whole idea of the PR is to not store `estimators_samples_` on the object any more it's not possible to use it for the consistency test.